packages/edge-login-ui-rn/src/common/locales/strings.js

```diff
@@ -39,9 +45,6 @@ export function selectLocale(locale: string = 'en'): boolean {
   let found = false
   const lang = normalizedLocale.slice(0, 2)
 
-  // Set default of US English
```

Comment:
Why was this removed?

Reply:
This is copy pasted from the react-gui strings.js to login-ui strings.js. So any changes are just replicated